ironfish-rust-nodejs/index.js

```diff
@@ -11,8 +11,7 @@ function isMusl() {
   // For Node 10
   if (!process.report || typeof process.report.getReport !== 'function') {
     try {
-      const lddPath = require('child_process').execSync('which ldd').toString().trim();
-      return readFileSync(lddPath, 'utf8').includes('musl')
+      return readFileSync('/usr/bin/ldd', 'utf8').includes('musl')
```

Comment:
I think this may need to be reverted, I'm pretty sure it was added as part of an upgraded napi-rs version